### DISCUSSION TITLE
Some questions about 《Definition of the Floating System for Phase IV of OC3》

### DISCUSSION THREAD

**Discussion by sjihf12** [OpenFAST/openfast#1743] [Q&A]
Dear Professor J. Jonkman,
Thank you for reading!
I have recently read 《Definition of the Floating System for Phase IV of OC3》, it is of great help to me. But I'm confused about something in it. 
1. In Table 4-2, why is there only four degrees of freedom of additional linear damping? The additional linear damping in roll and the additional linear damping in pitch is 0? If they're not 0, how to compute them?

2. In Table 5-1, does the additional yaw spring stiffness provided by the mooring system vary with floating platform movement? How to calculate the additional yaw spring stiffness provided by the mooring system?

**Comment by jjonkman** [comment]
Dear @sjihf12,

Here are my responses:

1. The "additional" linear damping for the OC3-Hywind spar were defined to provide damping not already accounted for through wave-radiation in the potential-flow solution and viscous drag in the strip-theory (Morison) solution.  The values were set to match free-decay behavior measured in tank testing.  No additional damping was needed in pitch and roll because the inherent damping from wave radiation and viscous drag was sufficient in pitch and roll.

2. The yaw spring is defined linearly, independent of floater motion.  The yaw spring was used because of the simplified representation of the mooring system defined for the OC3-Hywind model (including elimination of delta connection and replacement of a multisegmented lines with an equivalent homogeneous line).  The yaw spring was defined to provide the correct yaw restoring with the simplified mooring system to match the true behavior for the full mooring system.  You would not need to define the yaw spring if the full mooring system (including delta connection and multisegmented lines) is modeled directly.

Best regards,

**Reply by Melancho7ee** [reply]
Dear @jjonkman ,
How can I calculate the roll damping as you mentioned "wave-radiation in the potential-flow solution and viscous drag in the strip-theory (Morison) solution."? I intend to establish a model like your paper 《Influence of Control on the Pitch Damping of a Floating Wind Turbine》, where Equation 1 describe a 2-order simplified PtfmPitch motion. I would like to do something alike to describe the roll motion, and how can I obtain the socalled 
"$B_{radiation}+B_{viscous}$"?
![image](https://github.com/OpenFAST/openfast/assets/116873649/598e05b7-69a5-4e28-a181-6ae8d427f987)

**Reply by jjonkman** [reply]
Dear @Melancho7ee,

The paper you reference (https://www.nrel.gov/docs/fy08osti/42589.pdf) explains how B_Radiation and B_Viscous were calculated, i.e., B_Radiation was taken as the frequency-dependent radiation-damping matrix at the pitch natural frequency and B_Viscous was obtained through a FAST linearization analysis.  The former method (for B_Radiation) is likely OK, but OpenFAST also supports linearization of the potential-flow solution in terms of the state-space implemenation.  The latter (FAST linearization to obtain B_Viscous) is known to be inaccurate, unless the linearization is performed in the presence of current.  The latter can be improved through techniques such as stochastic viscous damping linearization, also known as Borgman linearization.  Improving how OpenFAST computes linearized viscous damping is a future development pathway, but not one that has been funded yet.

Best regards,

**Reply by Melancho7ee** [reply]
Dear @jjonkman ,
Thanks for your help! I have understood the methods to obtain the value of Roll Damping. It seems that the SS_Fitting MATLAB code can easily produce what I need.
